ccpp/physics_namelists/input_GFS_v16_RRTMGP.nml

```diff
@@ -1,148 +1,158 @@
-       iopt_crs       = 1
-       iopt_btr       = 1
-       iopt_run       = 1
-       iopt_sfc       = 1
-       iopt_frz       = 1
-       iopt_inf       = 1
-       iopt_rad       = 1
-       iopt_alb       = 2
-       iopt_snf       = 4
-       iopt_tbot      = 2
-       iopt_stc       = 1
-       debug          = .false.
-       oz_phys        = .false.
-       oz_phys_2015   = .true.
-       nstf_name      = 2,1,0,0,0
-       nst_anl        = .true.
-       psautco      = 0.0008,0.0005
-       prautco      = 0.00015,0.00015
-       lgfdlmprad     = .true.
-       effr_in        = .true.
-       ldiag_ugwp     = .false.
-       do_ugwp        = .false.
-       do_tofd        = .true.
-       do_sppt        = .false.
-       do_shum        = .false.
-       do_skeb        = .false.
-       do_RRTMGP      = .true.
-       active_gases   = 'h2o_co2_o3_n2o_ch4_o2'
-       ngases         = 6
-       rrtmgp_root    = '../../ccpp/physics/physics/rte-rrtmgp/'
-       lw_file_gas    = 'rrtmgp/data/rrtmgp-data-lw-g256-2018-12-04.nc'
-       lw_file_clouds = 'extensions/cloud_optics/rrtmgp-cloud-optics-coeffs-lw.nc'
-       sw_file_gas    = 'rrtmgp/data/rrtmgp-data-sw-g224-2018-12-04.nc'
-       sw_file_clouds = 'extensions/cloud_optics/rrtmgp-cloud-optics-coeffs-sw.nc'
-       doG_cldoptics  = .true.
-       rrtmgp_ngauss_ang = 3
-       rrtmgp_nrghice = 3
+    cal_pre = .false.
+    cdmbgwd = 4.0, 0.15, 1.0, 1.0
+    cnvcld = .true.
+    cnvgwd = .true.
+    debug = .false.
+    do_shum = .false.
+    do_skeb = .false.
+    do_spp = .false.
+    do_sppt = .false.
+    do_tofd = .true.
+    do_ugwp = .false.
+    dspheat = .true.
+    effr_in = .true.
+    fhcyc = 0
+    fhlwr = 1200.0
+    fhswr = 1200.0
+    fhzero = 1.0
+    h2o_phys = .true.
+    hybedmf = .false.
+    iaer = 5111
+    ialb = 1
+    iau_inc_files = ''
+    icliq_sw = 2
+    ico2 = 2
+    iems = 1
+    imfdeepcnv = 2
+    imfshalcnv = 2
+    imp_physics = 11
+    iopt_alb = 2
+    iopt_btr = 1
+    iopt_crs = 1
+    iopt_dveg = 1
+    iopt_frz = 1
+    iopt_inf = 1
+    iopt_rad = 1
+    iopt_run = 1
+    iopt_sfc = 1
+    iopt_snf = 4
+    iopt_stc = 1
+    iopt_tbot = 2
+    iovr = 3
+    isatmedmf = 1
+    isol = 2
+    isot = 1
+    isubc_lw = 2
+    isubc_sw = 2
+    ivegsrc = 1
+    ldiag3d = .true.
+    ldiag_ugwp = .false.
+    lgfdlmprad = .true.
+    lheatstrg = .true.
+    lndp_type = 0
+    lsm = 1
+    lsoil = 4
+    lwhtr = .true.
+    n_var_lndp = 0
+    n_var_spp = 0
+    nsradar_reset = 3600
+    nst_anl = .true.
+    nstf_name = 2, 1, 0, 0, 0
+    oz_phys = .false.
+    oz_phys_2015 = .true.
+    pdfcld = .false.
+    prautco = 0.00015, 0.00015
+    pre_rad = .false.
+    print_diff_pgr = .false.
+    prslrd0 = 0.0
+    psautco = 0.0008, 0.0005
+    qdiag3d = .true.
+    random_clds = .false.
+    redrag = .true.
+    satmedmf = .true.
+    shal_cnv = .true.
+    swhtr = .true.
+    trans_trac = .true.
+    use_ufo = .true.
+    do_RRTMGP          = .true.
```

Comment:
@ligiabernardet The only difference between the _RRTMGP nml's should be that do_RRTMGP=.true. Having two nml's results in the GP counterpart being left behind.

This is easier to manage in the UFS since do_rrtmgp is imported into the nml, so there is only a single nml to manage. https://github.com/ufs-community/ufs-weather-model/blob/develop/tests/parm/control.nml.IN#L189